python-phoenixdb/NEWS.rst

```diff
@@ -1,6 +1,10 @@
 Changelog
 =========
 
+Version 1.2.1
+-------------
+- Fixed flaky tests and defined authentication mechanism for SPNEGO explicitly (PHOENIX-6781)
```

Comment:
Nit:
You could separate out the SPNEGO fix to a separate line, as that's the point of the release.